frontend/DataView/Simple.js

```diff
@@ -200,6 +200,8 @@ function textToValue(txt) {
 function valueToText(value) {
   if (value === undefined) {
     return 'undefined';
+  } else if (value === Infinity) {
```

Comment:
Can we do `typeof value === 'number'` instead and cast to string? Then it will automatically cover `-Infinity` and `NaN` as well. (Though `-0` will still come out as `0`…)